controllers/memberoperatorconfig/mapper.go

```diff
@@ -1,43 +1,28 @@
 package memberoperatorconfig
 
 import (
-	"context"
-
-	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
 	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
 
 	corev1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/types"
 	ctrl "sigs.k8s.io/controller-runtime"
-	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/handler"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
-type SecretToMemberOperatorConfigMapper struct {
-	client client.Client
-}
+type SecretToMemberOperatorConfigMapper struct{}
 
 var _ handler.Mapper = SecretToMemberOperatorConfigMapper{}
 var mapperLog = ctrl.Log.WithName("SecretToMemberOperatorConfigMapper")
 
+// Map maps secrets to the singular instance of MemberOperatorConfig named "config"
 func (m SecretToMemberOperatorConfigMapper) Map(obj handler.MapObject) []reconcile.Request {
-	if secret, ok := obj.Object.(*corev1.Secret); ok {
+	if _, ok := obj.Object.(*corev1.Secret); ok {
 		controllerNS, err := k8sutil.GetWatchNamespace()
 		if err != nil {
 			mapperLog.Error(err, "could not determine watched namespace")
 			return []reconcile.Request{}
 		}
-
-		if secret.GetNamespace() != controllerNS {
```

Comment:
Yeah.. dropping namespace check should be fine since this client watches only the member operator namespace. We have another client for the all-namespace watchers which is currently used in the idler and status controllers only.